lib/src/Pos/Diffusion/Full/Block.hs

```diff
@@ -265,6 +273,140 @@ getBlocks logTrace logic recoveryHeadersMessage enqueue nodeId tipHeaderHash che
               Just (MsgBlock block) -> do
                   retrieveBlocksDo conv bvd (i - 1) (block : acc)
 
+streamBlocks
```

Comment:
Document

Reply:
Particularly the return type. Apparently `Nothing` means "fall back to batching"?

Reply:
But isn't it possible to transparently fall back to batching?

Reply:
Ok fair enough